### ISSUE TITLE
Demo doesn't run on Windows 10 x64 with Python 3.5.0

### ISSUE THREAD

**Issue by ibara** [hypatia-engine/hypatia#92]
From Slack:
```
Brian@kokoro MINGW64 ~
$ py --version
Python 3.5.0

Brian@kokoro MINGW64 ~/hypatia/demo (develop)
$ py game.py
Traceback (most recent call last):
  File "game.py", line 35, in <module>
    scene = game.Scene.from_tmx_resource('debug')
  File "C:\Users\Brian\AppData\Local\Programs\Python\Python35\lib\site-packages\hypatia_engine-0.2.30-py3.5.egg\hypatia\game.py", line 298, in from_tmx_resource
  File "C:\Users\Brian\AppData\Local\Programs\Python\Python35\lib\site-packages\hypatia_engine-0.2.30-py3.5.egg\hypatia\game.py", line 563, in __init__
  File "C:\Users\Brian\AppData\Local\Programs\Python\Python35\lib\site-packages\hypatia_engine-0.2.30-py3.5.egg\hypatia\tiles.py", line 85, in __init__
  File "C:\Users\Brian\AppData\Local\Programs\Python\Python35\lib\site-packages\hypatia_engine-0.2.30-py3.5.egg\hypatia\tiles.py", line 411, in from_resources
ValueError: invalid literal for int() with base 10: '0.5'
```

**Comment by lardnar** [comment]
I have win10x64 on my desktop at home, I can look into this by wednesday if nobody else can

**Comment by ibara** [comment]
With a fresh 3.5.0 install: :-(
```python
$ py -3 game.py
Traceback (most recent call last):
  File "game.py", line 35, in <module>
    scene = game.Scene.from_tmx_resource('debug')
  File "C:\Users\Brian\AppData\Local\Programs\Python\Python35\lib\site-packages\hypatia_engine-0.2.30-py3.5.egg\hypatia\game.py", line 308, in from_tmx_resource
  File "C:\Users\Brian\AppData\Local\Programs\Python\Python35\lib\site-packages\hypatia_engine-0.2.30-py3.5.egg\hypatia\game.py", line 594, in __init__
  File "C:\Users\Brian\AppData\Local\Programs\Python\Python35\lib\site-packages\hypatia_engine-0.2.30-py3.5.egg\hypatia\sprites.py", line 148, in __init__
  File "C:\Users\Brian\AppData\Local\Programs\Python\Python35\lib\site-packages\hypatia_engine-0.2.30-py3.5.egg\hypatia\resources.py", line 129, in __init__
  File "C:\Users\Brian\AppData\Local\Programs\Python\Python35\lib\site-packages\hypatia_engine-0.2.30-py3.5.egg\hypatia\resources.py", line 260, in load_gif
  File "C:\Users\Brian\AppData\Local\Programs\Python\Python35\lib\site-packages\hypatia_engine-0.2.30-py3.5.egg\hypatia\animatedsprite.py", line 603, in from_file
  File "C:\Users\Brian\AppData\Local\Programs\Python\Python35\lib\site-packages\hypatia_engine-0.2.30-py3.5.egg\hypatia\animatedsprite.py", line 701, in frames_from_gif
  File "C:\Users\Brian\AppData\Local\Programs\Python\Python35\lib\site-packages\hypatia_engine-0.2.30-py3.5.egg\hypatia\animatedsprite.py", line 750, in pil_image_to_pygame_surface
  File "C:\Users\Brian\AppData\Local\Programs\Python\Python35\lib\site-packages\pillow-3.0.0-py3.5-win-amd64.egg\PIL\Image.py", line 686, in tostring
Exception: tostring() has been removed. Please call tobytes() instead.
```

**Comment by lillian-lemmer** [comment]
@ibara It happens because I stupidly required a specific version of Pillow, so when Python 3.5 rolled around, it broke Pillow. An issue resolved by using a newer version of Pillow.

Right now I'm dealing with:

> assert frame[1] > 0, 'Frame %s duration must be greater than zero.' % (i)
AssertionError: Frame 0 duration must be greater than zero.

Running Python 3.5 in Windows 10.

**Comment by lillian-lemmer** [comment]
@ibara It happens because I stupidly required a specific version of Pillow, so when Python 3.5 rolled around, it broke Pillow. An issue resolved by using a newer version of Pillow.

Running Python 3.5 in Windows 10.

I'm working on this now.

**Comment by ibara** [comment]
@lillian-lemmer kk. No worries! Happy to test when you're ready for me to.

**Comment by lillian-lemmer** [comment]
Resolved in cbc14bb!

**Comment by ejmr** [comment]
Useful tip---which unfortunately forgot to mention when updating the project guidelines re. Git---if a commit message contains a line like...

```
Fixes: #92
Resolves: #92
Closes: #92
```

...then GitHub will automatically close that issue when you push the commit.  Useful for indicating which issue a patch addresses and closing said issue at the same time.

**Comment by lillian-lemmer** [comment]
@ejmr that's great to know thank you!
